GetJobber/atlantis
# feat(components-native): Import InputCurrency

## Motivations



## Changes



### Added

- feat(components-native): Added InputCurrency
- feat(components-native): Added currencySymbol to AtlantisContext

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- run `npm run pack @jobber/components-native`
- Install into Mobile project
- Replace InputCurrency with the new InputCurrency and verify it works

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
